docs/Makefile

```diff
@@ -1,29 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
In general, I feel like we should add the docs/* to the .gitignore. That being said though, why delete this one? Was it causing clashes with the versioner? I am asking as I have a feeling removing this may make Sphinx unusable

Reply:
I removed most pyscafflold artifacts from the repo, we can use some of the tools it uses and Sphinx can be one of them. We can add as a ticket for this later if we want to